agent/browser/ie/wptbho/wpt.cc

```diff
@@ -73,7 +74,7 @@ LPOLESTR GET_USER_TIMINGS = L"wptGetUserTimings";
 LPOLESTR GET_NAV_TIMINGS = L"wptGetNavTimings";
 LPOLESTR GET_LOAD_EVENT_END_TIMING = L"wptGetLoadEventEndTiming";
 
-int MAX_ONLOAD_WAIT = 20; // 4s
+int MAX_ONLOAD_WAIT = 4; // 4s

```

Comment:
Yes, it's still valid. It's a safety net.
On Mon, May 23, 2016 at 13:22 Karthik Krishnamurthy <
notifications@github.com> wrote:

> In agent/browser/ie/wptbho/wpt.cc
> <https://github.com/Appdynamics/webpagetest/pull/131#discussion_r64281921>
> :
>
> > @@ -73,7 +74,7 @@ LPOLESTR GET_USER_TIMINGS = L"wptGetUserTimings";
> >  LPOLESTR GET_NAV_TIMINGS = L"wptGetNavTimings";
> >  LPOLESTR GET_LOAD_EVENT_END_TIMING = L"wptGetLoadEventEndTiming";
> >
> > -int MAX_ONLOAD_WAIT = 20; // 4s
> > +int MAX_ONLOAD_WAIT = 4; // 4s
>
> Is the comment '// 4s' still valid?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Appdynamics/webpagetest/pull/131/files/5c20818c26d05523b6088cba81140dcf828eb5a0#r64281921>
>